resources/puppetlabs/lein-ezbake/template/foss/ext/redhat/ezbake.service.erb

```diff
@@ -22,19 +23,12 @@ PermissionsStartOnly=true
 ExecStartPre=<%= action %>
 <% end -%>
 <% end -%>
+
 ExecReload=/opt/puppetlabs/server/apps/<%= EZBake::Config[:real_name] %>/bin/<%= EZBake::Config[:real_name] %> reload
-ExecStart=/usr/bin/java $JAVA_ARGS \
-          '-XX:OnOutOfMemoryError=kill -9 %%p' \
-          -Djava.security.egd=/dev/urandom \
-          -cp "${INSTALL_DIR}/<%= EZBake::Config[:uberjar_name] %>" clojure.main \
-          -m <%= EZBake::Config[:main_namespace] %> \
-          --config "${CONFIG}" \
-          -b "${BOOTSTRAP_CONFIG}" $@
+ExecStart=/opt/puppetlabs/server/apps/<%= EZBake::Config[:real_name] %>/bin/<%= EZBake::Config[:real_name] %> start
 
 KillMode=process
 
-ExecStartPost=/bin/bash "${INSTALL_DIR}/ezbake-functions.sh" wait_for_app
```

Comment:
This post should go away, as the start script removes the need for it. The generic templated list immediately below should stay.